_examples/hi/test.py

```diff
@@ -15,20 +15,20 @@
 print("--- hi.Universe:", hi.Universe)
 print("--- hi.Version:", hi.Version)
 
-print("--- hi.Debug():",hi.Debug())
+print("--- hi.Debug():", hi.Debug())
```

Comment:
could you factor out all these cosmetic changes (that I generally agree with) into a dedicated commit ?
(it will be easier for diffs.)